### PR TITLE
Uid_map and GridLayout

### PR DESCRIPTION
**Pull request by mischa99**
- shows set result variable in GUI after loading a dialog from file (uses uid_map key as value of the result var in XML document, instead of variable name) 
-> change of #readAttribute needed, to fix this in other Plugins
- this solution is backwards compatible: if user opens file saved under old version and saves the result variable in GUI one extra time, the variable is shown the next time when the dialog is openend from file

- GridBagLayout in GUI for the JPanels

### CODE REVIEW

File: src/main/java/edu/cmu/lti/dialogos/db/sqlite/SqliteNode.java [hunk 1]
```diff
@@ -132,10 +133,10 @@ public void readAttribute(XMLReader r, String name, String value, IdMap uid_map)
         if (name.equals(RESULT_VAR) && value!=null) {
             try {
                 this.setProperty(name, uid_map.variables.get(value));
-
             } catch (Exception exn) {
-                r.raiseException(com.clt.diamant.Resources.format("UnknownVariable", "ID " + value));
-           }
+                this.setProperty(name, value);
+//                r.raiseException(com.clt.diamant.Resources.format("UnknownVariable", "ID " + value));
```
Comment (timobaumann) [inline src/main/java/edu/cmu/lti/dialogos/db/sqlite/SqliteNode.java]: please remove this commented out line.

File: minimal-example.dos [hunk 2]
```diff
@@ -5,6 +5,11 @@
 	<plugin type="dialogos.plugin.sqlite">
 		<att name="dbURL" type="string" value="jdbc:sqlite:millionaire.db"/>
 	</plugin>
+	<plugin type="dialogos.plugin.tts">
+	</plugin>
+	<plugin type="dialogos.plugin.sphinx">
+		<att name="language" type="string" value="de_DE"/>
+	</plugin>
```
Comment (timobaumann) [inline minimal-example.dos]: can you check this in without adding these 5 lines?

**Comment by timobaumann** [comment]
Hi Mikhail, can you please remove all the extra files (.DS_Store, a bunch of savexyz.xml files, some dos models you used for testing)? Please also have another look at the comments that I added in the "files changed" view.

**Comment by mischa99** [comment]
Sure! Thanks for the hint.

**Pull request merged**